tests/integration/test_xlsx_regression.py

```
@@ -0,0 +1,172 @@
+    assert after["merges"] == before["merges"]
+    assert after["dimensions"] == before["dimensions"]
+    assert after["styles"] == before["styles"]
+
+
+@pytest.mark.parametrize("kind", ["empty", "incomplete"])
+def test_empty_or_incomplete_model_response_is_rejected(tmp_path, kind):
+    def response(texts):
+        complete = [f"Перевод {i}" for i, _ in enumerate(texts)]
+        return [] if kind == "empty" else complete[:-1]
+    output = tmp_path / f"{kind}.xlsx"
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)